tests/test_F0_output.py

```diff
@@ -0,0 +1,189 @@
+
+
+# def test_plot_energy_flows_limit_to_14_days():
+#    assert 0 == 0
+
+
+# def test_plot_energy_flows_limit_to_365_days():
+#    assert 0 == 0
+
+
+class TestFileCreation:
+    def setup_class(self):
+        os.mkdir(OUTPUT_PATH)
+
+    def test_store_barchart_for_capacities(self):
+        dict_scalar_capacities = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "project_data": {
+                "project_name": "a_project",
+                "scenario_name": "a_scenario",
+            },
+            "kpi": {
+                "scalar_matrix": pd.DataFrame(
+                    {"label": ["asset_a", "asset_b"], "optimizedAddCap": [1, 2]}
+                )
+            },
+        }
+        F0.plot_optimized_capacities(dict_scalar_capacities)
+        assert (
+            os.path.exists(
+                os.path.join(OUTPUT_PATH, "optimal_additional_capacities.png")
+            )
+            is True
+        )
+
+    def test_store_scalars_to_excel_two_tabs_dict(self):
+        dict_scalars_two_tabs_dict = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "kpi": {
+                "economic": pandas_Dataframe,
+                "technical": {"param1": 1, "param2": 2},
+            },
+        }
+        F0.store_scalars_to_excel(dict_scalars_two_tabs_dict)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "scalars" + ".xlsx")) is True
+
+    def test_store_scalars_to_excel_two_tabs_no_dict(self):
+        dict_scalars_two_tabs = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "kpi": {"economic": pandas_Dataframe, "technical": pandas_Dataframe},
+        }
+
+        F0.store_scalars_to_excel(dict_scalars_two_tabs)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "scalars" + ".xlsx")) is True
+
+    def test_store_each_bus_timeseries_to_excel_and_png_one_bus(self):
+        dict_timeseries_test_one_bus = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "optimizedFlows": {"a_bus": bus},
+        }
+
+        F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_one_bus)
+        assert (
+            os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses" + ".xlsx"))
+            is True
+        )
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "a_bus" + " flows.png")) is True
+
+    def test_store_each_bus_timeseries_to_excel_and_png_two_busses(self):
+        dict_timeseries_test_two_busses = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "optimizedFlows": {"a_bus": bus, "b_bus": bus},
+        }
+        F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_two_busses)
+        assert (
+            os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses" + ".xlsx"))
+            is True
+        )
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "a_bus" + " flows.png")) is True
```

Comment:
```suggestion
        assert os.path.exists(os.path.join(OUTPUT_PATH, "a_bus flows.png")) is True
```

Reply:
This is not the only occurence, if you want I can check all of them

Reply:
I would rather like to leave it like this, there might be further plans I have for this ;)